waspc/data/Generator/templates/sdk/wasp/package.json

```diff
@@ -22,6 +22,8 @@
       {=!  Used by our code, uncodumented (but accessible) for users. =}
       "./auth/session": "./dist/auth/session.js",
       {=!  Used by our code, uncodumented (but accessible) for users. =}
+      "./auth/user": "./dist/auth/user.js",
+      {=!  Used by our code, uncodumented (but accessible) for users. =}
```

Comment:
Don't forget to add to the table of visible API (I know it's a pain :grimacing:).

Also, please specify which part of our code needs it (i.e., framework or sdk). I know most exports don't specify this, but doing it for new stuff can't hurt (unless it's wrong, of course, but it won't be :smile:).

Reply:
Sure thing, I updated the Sheet a bit, but not all the way 👍  

I'll update the comments as well.

Reply:
Updated the comments to say private API exports are used in SDK and server